web/src/components/core/ChangeProductLink.test.tsx

```diff
@@ -30,11 +30,13 @@ const tumbleweedProduct = {
   id: "Tumbleweed",
   name: "openSUSE Tumbleweed",
   description: "Tumbleweed description...",
+  icon: "Tumbleweed.icon",
```

Comment:
icon suffix?

Reply:
I'm not in favor of four letter suffixes, plus the format is svg.